keras_eval/visualizer.py

```diff
@@ -263,3 +265,58 @@ def plot_concept_metrics(concepts, metrics, x_axis_label, y_axis_label, title=No
     )
     fig = go.Figure(data=data, layout=layout)
     iplot(fig, filename='line-mode')
+
+
+def plot_models_performance(eval_dir, individual=False, class_idx=None, metric=None, save_name=None):
+    '''
+       Enables plotting of multiple models metrics for comparison
+       Args:
+           eval_dir: A directory that contains multiple metric files
+           individual: If True,compare individual metrics else, average metrics
+           class_idx: The index of class for when comparing individual metrics
+           metric: The metric to be plotted for comparison
+           save_name: The name, location of where the plot is to be saved
+
+       Returns: Nothing, Plot can be saved.
+
+       '''
+    x_axis = []
+    y_axis = []
+    tick_label = []
+    i = 0
+    for result_csv in os.listdir(eval_dir):
+        if check_result_type(result_csv, individual):
+            df = pd.read_csv(os.path.join(eval_dir, result_csv))
+            tick_label.append(result_csv[:result_csv.rfind('_')])
+            if individual:
+                if isinstance(class_idx, int) and isinstance(metric, str):
+                    y_axis.append(df[metric][class_idx])
+                    x_axis.append(i)
+                else:
+                    raise ValueError('Missing required option(s): class_idx, metric')
+            else:
+                if metric:
```

Comment:
Be careful here:
do you want to check it that way?
https://amir.rachum.com/blog/2012/08/25/you-cant-handle-the-truth/